README.md

```diff
@@ -33,39 +33,74 @@ HTTP/HTTPS and S3 sources are currently supported.
 
 The following are some environment variables you'll need to specify to have this running.
 
+*unix
```

Comment:
I would use the term Linux, more explicitly, as appu is not tested on other UN*Xes.
Also the Markdown format you chose is different from the Windows section. I would make them the same and not a list item. Maybe `##### Linux` and `##### Windows (CMD)`. Or even, just text, like `For Linux:` and `For cmd.exe on Windows:`.